cmd/wigStats/wigStats.go

```diff
@@ -0,0 +1,80 @@
+// Command Group: "Wig Tools"
+//TODO make wigStats compatible with negative wig values
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/wig"
+	"log"
+	"math"
+)
+
+func wigStats(inFile string, noGapFile string, outFile string, missingDataValue float64) {
+	var records []bed.Bed = bed.Read(noGapFile)
+	var w []wig.Wig = wig.Read(inFile)
+	var wigValues []float64
+	var statValues, tmpValues []int
+	var err error
+	var i, j int
+	statValues = make([]int, 1000)
+	for i = range records {
+		wigValues = wig.ChromToSlice(w, records[i].Chrom)
+		for j = records[i].ChromStart; j < records[i].ChromEnd; j++ {
+			if wigValues[j] == missingDataValue {
+				continue
+			}
+			if int(wigValues[j]) > len(statValues) {
```

Comment:
Did we decide if this line needs to be changed to int(wigValues[j]) >= len(statValues)?

Reply:
Yes, I think this is the last change that we need before merging.